CMakeLists.txt

```diff
@@ -154,6 +159,18 @@ if(BUILD_PYTHON)
 
     set(PYTHON_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR}/python-build)
 
+    # python setup.py install seems to behave differently if prefix
+    # is supplied, requiring setting pythonpath after install.
+    # If a custom prefix is not set, don't supply prefix at allocate_lock
+    # /usr/local is the default on linux.
+    # Install not really tested on windows as of yet.
+    if(${CMAKE_INSTALL_PREFIX} STREQUAL "/usr/local")
+        set(PYTHON_PREFIX_LINE, "")
```

Comment:
I don't think that comma's meant to be there - otherwise looks good.

Reply:
Fixed